packages/sanity/src/core/form/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -317,7 +320,6 @@ export const DefaultBlockObjectComponent = (props: BlockProps) => {
           actions: (
             <BlockObjectActionsMenu
               focused={focused}
-              isActive
```

Comment:
This is always true. Was a variable previously. Remove it and use `focused` prop inside `BlockObjectActionsMenu` instead.